python/lsst/ci/git.py

```diff
@@ -26,7 +26,8 @@ def __call__(self, *args, **kwargs):
 
         return_status = kwargs.get("return_status", False)
 
-        cmd = ('git',) + args
+        # force all cli args into strings
+        cmd = ['git'] + [str(x) for x in args]
```

Comment:
I have but I'm actively avoiding major refactoring of this code because it has no unit tests.  Eups is not pip install-able so a proper test harness would require some thought.